packages/amplify-app/src/index.js

```diff
@@ -168,6 +168,7 @@ async function guessPlatform() {
     javascript: 'amplify-frontend-javascript',
     android: 'amplify-frontend-android',
     ios: 'amplify-frontend-ios',
+    flutter: 'amplify-frontend-flutter',
```

Comment:
amplify-app needs a dependency on this new package

Reply:
@attilah Do we have a way to easily disable particular commands?  It would be nice to just shut off api, app, codegen, etc until those features are supported.

Reply:
We are not supporting amplify-app yet.  This is just a developer preview.